src/image.rs

```diff
@@ -672,6 +673,7 @@ pub trait GenericImage: GenericImageView {
     /// Puts a pixel at location (x, y)
     ///
     /// This function can be implemented in a way that ignores bounds checking.
+    #[allow(clippy::missing_safety_doc)]
```

Comment:
Shouldn't similar warnings appear in `ImageBuffer`? If so then please add the safety sections there as well.

Reply:
I've the added safety section as well!!